### PR TITLE
[GR-42167] Add `-H:±RingBellWhenDone` option.

### PR DESCRIPTION
**Pull request by graalvmbot**
Add `-H:±RingBellWhenDone` option to allow users receive audible notifications when Native Image builder is done.

### CODE REVIEW

File: substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java [hunk 1]
```diff
@@ -740,8 +739,24 @@ private static Timer getTimer(TimerCollection.Registry type) {
...
     private static void resetANSIMode() {
-        NativeImageSystemIOWrappers.singleton().getOut().print(ANSI.RESET);
+        var out = NativeImageSystemIOWrappers.singleton().getOut();
+        out.print(ANSI.RESET);
+        out.flush();
+    }
+
+    private static void ringTerminalBell() {
+        var out = NativeImageSystemIOWrappers.singleton().getOut();
+        out.write(0x07);
```
Comment (olpaw) [inline substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java:758]: Does this work on Windows?

**Comment by olpaw** [comment]
I know this is kinda fun, but do we really need this?

If I'm interested in the completion of a long running process I instead use e.g.
```bash
native-image -jar MyHugeApp.jar && echo -e '\a'
```
or even better
```bash
native-image -jar MyHugeApp.jar && zenity --info --text="THE BUILD IS DONE"
```

**Comment by fniephaus** [comment]
Agreed, let's make sure this is properly documented instead.

File: substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java (hunk 1 above)
Comment (fniephaus) [reply substratevm/src/com.oracle.svm.hosted/src/com/oracle/svm/hosted/ProgressReporter.java:758]: Yes, at least the command prompt supports the bell character.

**Pull request closed without merging**